### ISSUE TITLE
Modified .po files are not updated when running a game with USB remote debugging

### ISSUE THREAD

**Issue by owlnewworlds** [godotengine/godot#99566]
### Tested versions

Reproducible in Godot 4.3

### System information

Godot v4.3.stable - macOS 15.1.0 - Vulkan (Mobile) - integrated Apple M3 Pro - Apple M3 Pro (12 Threads)

### Issue description

When using .po files to translate your game and testing the game with USB remote debugging, it looks like Godot is using the last installed .po files and not the last updated ones. You need to remove the app from your phone and run in again to see the updates. When you run through the editor on your computer, all the data are up-to-date as expected.

### Steps to reproduce

You must have a game with the translations enabled and using .po files.
Switch from Godot to the external text editor to update any of the .po files.
Run your game on a device with USB remote debugging
The game displays old values of the .po files and not the most recent .po files
- It only works if you remove the app from your device and run again with USB remote debugging
- No issue when running game through the editor on your computer

### Minimal reproduction project (MRP)

N/A

**Comment by Nallebeorn** [comment]
Not able to reproduce when running a game on Android (Google Pixel 8a, Android 15) from Windows 11 (tested 4.3-stable and 4.4.1-stable).

@owlnewworlds In your case, were you running on an iPhone?

Test project I used: [po-update-usb-debugging.zip](https://github.com/user-attachments/files/19816647/po-update-usb-debugging.zip). If I update the msgstr value in translations.po and redeploy on my Android device, I see the latest translation value.

cc @AThousandShips

**Comment by owlnewworlds** [comment]
> Not able to reproduce when running a game on Android (Google Pixel 8a, Android 15) from Windows 11 (tested 4.3-stable and 4.4.1-stable).
> 
> [@owlnewworlds](https://github.com/owlnewworlds) In your case, were you running on an iPhone?
> 
> Test project I used: [po-update-usb-debugging.zip](https://github.com/user-attachments/files/19816647/po-update-usb-debugging.zip). If I update the msgstr value in translations.po and redeploy on my Android device, I see the latest translation value.
> 
> cc [@AThousandShips](https://github.com/AThousandShips)

I was running it from an Android device, not iPhone.

What do you mean by redeploy? In my case I was pressing the "remote deploy" button in the editor. I dont know if its triggering a full redeploy of the app (with the updated .po files) hence my issue.

Maybe it's been fixed by another fix in Godot 4.4?
I'll check again.

**Comment by Nallebeorn** [comment]
>> What do you mean by redeploy? In my case I was pressing the "remote deploy" button in the editor. I dont know if its triggering a full redeploy of the app (with the updated .po files) hence my issue.
> 
> Maybe it's been fixed by another fix in Godot 4.4? I'll check again.

Yes, the "Remote deploy" button is what I used. I didn't need to go in to my phone and uninstall anything to make it work, the translations were updated just as if I had done "Run Project" to run locally.
